### DISCUSSION TITLE
Why isn't VuePress archived?

### DISCUSSION THREAD

**Discussion by nbro** [vuejs/vitepress#3747] [General]
I've started to learn and use Vue a couple of weeks ago and I've also come across VuePress. But I've realised that it doesn't really allow you to customise anything. For example, it seems to force you to put the markdown files under `docs`, while I just want to turn my markdown notes into a site as quickly as possible without changing any structure. I don't want to move my markdown files into `docs`. Anyway, I've also realised that VuePress is no longer developed, so why don't you just add a very clear notice to the beginning of the README there or maybe just archive the repo?

**Comment by brc-dd** [comment]
>  so why don't you just add a very clear notice to the beginning of the README there

https://github.com/vuejs/vuepress#status

**Reply by nbro** [reply]
Yes, I should have said "why don't you add a clear notice to the VuePress landing page": https://vuepress.vuejs.org/. Indeed, I didn't look at the README but only the docs and I don't remember any notice there, but it's possible I missed it. Also, that markdown style is, imo, not very clear. It should be bold.

**Reply by nbro** [reply]
https://github.com/vuejs/vuepress/pull/3212
